sources/packages/web/src/views/aest/StudentApplicationView.vue

```diff
@@ -95,6 +96,9 @@ export default defineComponent({
       selectedForm.value = applicationDetail.value.applicationFormName;
       initialData.value = {
         ...applicationDetail.value.data,
+        applicationOfferingIntensity: mapOfferingIntensity(
+          applicationDetail.value.applicationOfferingIntensity,
+        ),
```

Comment:
In a long term these translations must be component responsibility and hence translated inside the component. 

For this PR, I am not going to ask the change as requires little extra work and we are at end of sprint. So calling it not a blocker.